crates/re_viewer_context/src/gpu_bridge/re_renderer_callback.rs

```diff
@@ -78,7 +78,7 @@ impl egui_wgpu::CallbackTrait for ReRendererCallback {
 
     fn paint<'a>(
         &'a self,
-        info: egui::PaintCallbackInfo,
+        _info: egui::PaintCallbackInfo,
```

Comment:
So the viewport coming from egui was somehow wrong?
Is this a bug in an egui crate?

Reply:
we only passed on the viewport offset from egui, not the viewport size. We then assumed the size was what we had previously on the viewbuilder.
... which come to think means that the viewbuilder got an outdated size! This is concerning but I reckon this can easily happen though when new sizes come in asynchronously; to nail this down perfectly we'd need to know when egui queries the size that leads to the final renderpass.
This PR instead just trusts egui with the viewport setting, after all egui also set up the renderpass and knows what a valid viewport is for that renderpass.